modules/pages/client/components/Home.component.js

```diff
@@ -99,7 +85,7 @@ export default function Home({ user, isNativeMobileApp, photoCredits }) {
 
   const boards = getBoardPictures(circleRoute);
 
-  const [circles, setCircles] = useState([]);
+  const [, setCircles] = useState([]);
```

Comment:
is there a reason to use `[, setCircles]` compared to `[setCircles]`?

Reply:
it was the auto fix suggestion of VSC ;)